common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -441,6 +519,214 @@ public void run() {
         });
     }
 
+    @Override
+    public @NonNull
+    String sign(@NonNull final String alg,
+                @NonNull final String input) throws ClientException {
+        final Exception exception;
+        final String errCode;
+
+        try {
+            final byte[] inputBytesToSign = input.getBytes(ENCODING_UTF8);
+            final KeyStore.Entry keyEntry = mKeyStore.getEntry(KEYSTORE_ENTRY_ALIAS, null);
+
+            if (!(keyEntry instanceof KeyStore.PrivateKeyEntry)) {
+                Logger.warn(
+                        TAG + ":sign",
```

Comment:
may be store `:sign` in a var called methodName to follow the pattern in other places

Reply:
Applied in ef84e4ac69a53770a901b7fa2498c1bcff58790f thanks!